src/chi-vue/src/components/column-customization/ColumnCustomization.tsx

```diff
@@ -173,9 +173,22 @@ export default class ColumnCustomization extends Vue {
     const modal = this._modal();
 
     return (
-      <div class={DATA_TABLE_CLASSES.COLUMNS}>
-        {modalButton}
-        {modal}
+      <div class={DATA_TABLE_CLASSES.ACTIONS}>
+        <div class={`${DATA_TABLE_CLASSES.ACTIONS}-desktop`}>
+          {modalButton}
+          {modal}
+        </div>
+        <div class={`${DATA_TABLE_CLASSES.ACTIONS}-mobile`}>
+          <button
+            class="chi-button -portal -icon -primary -flat chi-drawer__trigger"
```

Comment:
classes.ts provides all the classes you use here.
Can you import `BUTTON_CLASSES`and update the classes, please?